spring-boot/admin/src/main/java/eu/coatrack/admin/DatabaseInitializer.java

```diff
@@ -395,7 +395,7 @@ private ApiKey createNewApiKey(
 
         apiKey = apiKeyRepository.save(apiKey);
 
-        serviceApi = serviceApiRepository.findOne(serviceApi.getId());
+        serviceApi = serviceApiRepository.findById(serviceApi.getId()).orElse(null);
```

Comment:
Instead of the orElse (null), it would be better if a specific exception is throwed here, with a message for example "The service API with the [id] wasn't found on the database".

Reply:
Good point. Done. I just added a NotFoundException without any message as I think this is self-explanatory.

Reply:
Due to @gr-hovest-atb remark I removed these exceptions again.

Reply:
Yes, I will create a issue for it so that we keep it separated